### PR TITLE
55 event scheduling

### LINKED ISSUE #55: Viewing Party: Trying to schedule in Past
User should not be allowed to schedule an event for a date/time in the past

### PR DESCRIPTION
**Pull request by dunlapww**
closes #55

### CODE REVIEW

File: app/models/viewing_party.rb [hunk 1]
```diff
@@ -9,4 +9,6 @@ class ViewingParty < ApplicationRecord
   validates :duration, presence: true
   validates :date, presence: true
   validates :time, presence: true
+  validates :date, inclusion: {in: (Date.today..Date.today+5.years)}
```
Comment (GeorgieGirl24) [inline app/models/viewing_party.rb:12]: Wow! Nice job here!

File: spec/models/viewing_party_spec.rb [hunk 2]
```diff
@@ -14,5 +14,35 @@
...
+      )
+      movie = Movie.create(title: 'fight club', uuid: 550)
+
+      vp = user.viewing_parties.new(movie_id: movie.id, duration: 100, date: '2018-01-01', time: Time.now, created_at: Time.now, updated_at: Time.now)
+      expect(vp.save).to eq(false)
+      vp = user.viewing_parties.new(movie_id: movie.id, duration: 100, date: Date.today, time: Time.now, created_at: Time.now, updated_at: Time.now)
+      expect(vp.save).to eq(true)
+    end
+
+    it 'should only accept duration >= 5' do
```
Comment (GeorgieGirl24) [inline spec/models/viewing_party_spec.rb:32]: Is this a time limit of 5 hours for the party? Was that just arbitrary ...

**Review by GeorgieGirl24 — APPROVED** [review]

File: app/models/viewing_party.rb (hunk 1 above)
Comment (katemorris) [reply app/models/viewing_party.rb:12]: this is awesome!

**Review by katemorris — APPROVED** [review]
Looks great!

File: spec/models/viewing_party_spec.rb (hunk 2 above)
Comment (dunlapww) [reply spec/models/viewing_party_spec.rb:32]: 5 minutes, but yes a fairly arbitrary number.   Just want to make sure it's not negative.

**Pull request merged**